components/TextFields/BUILD

```diff
@@ -20,6 +20,21 @@ load("@build_bazel_rules_apple//apple:swift.bzl", "swift_library")
 
 licenses(["notice"])  # Apache 2.0
 
+mdc_objc_library(
+    name = "private",
+    hdrs = native.glob(["src/private/*.h"]),
+    includes = ["src/private"],
+    visibility = [":test_targets"],
+)
+
+package_group(
+    name = "test_targets",
+    packages = [
+        "//components/Chips/...",
```

Comment:
Can this be scoped to just the test targets of the components?

Reply:
It looks like both `visibility` and `package_group` require packages, not target names:

https://docs.bazel.build/versions/master/be/common-definitions.html
https://docs.bazel.build/versions/master/be/functions.html#package_group

Do you know of another way to do this? I think the only way is to move the tests into their own package.

Reply:
Ah bummer. I don’t know of another way unfortunately. I think it’s ok for us to rely on code review catching imports of the testing header rather than refactoring the test targets right now. Lgtm!